js/modules/k6/grpc/client.go

```diff
@@ -384,10 +384,18 @@ func resolveServiceFileDescriptors(
 	return fdset, nil
 }
 
+// sendReceiver is a smaller interface for decoupling
+// from reflectpb.ServerReflection_ServerReflectionInfoClient
+// that has the dependency from grpc.ClientStream.
+type sendReceiver interface {
+	Send(*reflectpb.ServerReflectionRequest) error
+	Recv() (*reflectpb.ServerReflectionResponse, error)
+}
```

Comment:
This is basically a mock for `ServerReflection_ServerReflectionInfoClient` for tests, right? Could you please update the comment to reflect that specific point, about mock & tests? Right now it sounds a tad convoluted (and therefore, meaning might be lost in the future) :sweat_smile:

Reply:
Yes, mock is the concrete action in this case, but generally I don't think it's a good thing have a so big dependency with no reason. Tried to improve the comment in the fixup.